### ISSUE TITLE
Boat demo

### ISSUE THREAD

**Issue by tomlogan501** [Intelligent-Quads/iq_sim#5]
Hi,
I test your simulation with the boat and it pops 2 boats and sinks them xD.

Is there a configuration to enable the USV plugin ? I can also use the VRX repository if needed.

Do you use the ardupilotplugin in Gazebo ? I can see in the code you try ArdupilotCopter plugin (on boat branch).

I would like to test it with one of our boat (same size as WAMV)

**Comment by ericjohnson97** [comment]
My simulation is built on the USV plugin which is found within the px4 sitl_gazebo repository. make sure to follow these instructions here 
https://github.com/Intelligent-Quads/iq_sim/blob/master/docs/boat_setup.md#6-setup-libgazebo_usv_dynamics_plugin

and also make sure to add `export GAZEBO_PLUGIN_PATH=${GAZEBO_PLUGIN_PATH}:$HOME/src/sitl_gazebo/build` to your bashrc and to source it for use in the same terminal.

**Comment by tomlogan501** [comment]
Seems that I didn't replace the plugin path
Now it works quite well

![image](https://user-images.githubusercontent.com/56969577/91017839-08743780-e5ef-11ea-8b2f-23d36981cb89.png)

Do you had any PID parms ?

**Comment by ericjohnson97** [comment]
to be completely honest. I've put the boat project on hold for the time being, and I didn't do too much tuning. I was able to get to track much better after turning down the speed and, tweaking a few other params I found on this page. https://ardupilot.org/rover/docs/rover-tuning-steering-rate.html. I never, got it perfect, but it was ok. next time I boot up my ubuntu side of my desktop I can take a look for you

**Comment by tomlogan501** [comment]
Yeah the link you provide is the basic tuning I use most of the time.
Tell me when you switch to the ubuntu "good" side :)

**Comment by Yeah2333** [comment]
@ericjohnson97 @tomlogan501 Hi,i test boat setup. When launch gazebo,  the boat sinks.how to fix it?
